### PR TITLE
Add 'Pony Town' and 'Ashes Town' presence

### PR DESCRIPTION
**Pull request by ponfertato**
Снимок экрана 2021-07-12 190123
Снимок экрана 2021-07-12 190058
Pony Town Official Server

Снимок экрана 2021-07-12 193109
Снимок экрана 2021-07-12 193151
Ashes Town Custom Server

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codefactor and schema errors.

File: websites/P/Pony Town/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "Pony Town",
+  "author": {
+    "name": "Ponfertato",
+    "id": "215527730345803777"
+  },
+  "category": "games",
+  "logo": "https://pony.town/assets/images/logo-small-770b141d9d.png",
+  "thumbnail": "https://pony.town/assets/images/logo-large-57d9b1947a.png",
```
Comment (Bas950) [inline websites/P/Pony Town/dist/metadata.json:10]: upload to imgur

File: websites/A/Ashes Town/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "Ashes Town",
+  "author": {
+    "name": "Ponfertato",
+    "id": "215527730345803777"
+  },
+  "category": "games",
+  "logo": "https://ashes.town/assets/images/logo-small-0decfb2e2f.png",
+  "thumbnail": "https://ashes.town/assets/images/logo-large-a04e72411e.png",
```
Comment (Bas950) [inline websites/A/Ashes Town/dist/metadata.json:10]: upload to imgur

File: websites/A/Ashes Town/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  "version": "0.0.1",
```
Comment (Bas950) [inline websites/A/Ashes Town/dist/metadata.json:11]:
```suggestion
  "version": "1.0.0",
```

File: websites/P/Pony Town/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  "version": "0.0.1",
```
Comment (Bas950) [inline websites/P/Pony Town/dist/metadata.json:11]:
```suggestion
  "version": "1.0.0",
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
There is not a lot to these presences, try adding some more stuff if possible.

**Comment by Ponfertato** [comment]
All Done, Problems solved

**Comment by SlowLife1661** [comment]
Hey @Ponfertato,
Unfortunately, your pull request has been **denied**, due to the following reason(s):
* **Low quality** presence
  * Not reading any DOMs
  * Only tell which page user is viewing and nothing more

**Pull request closed without merging**